packages/help-center/src/happychat-window-communicator.ts

```diff
@@ -16,8 +24,13 @@ export function useHCWindowCommunicator(
 		};
 	} );
 
-	const supportSite = selectedSite || userDeclaredSite;
-	const { resetPopup, resetStore } = useDispatch( STORE_KEY );
+	const currentSite = useSelect( ( select ) =>
+		select( 'automattic/site' ).getSite( window._currentSiteId )
```

Comment:
Could the store be referenced as `import { Site } from '@automattic/data-stores'` and `Site.STORE_KEY`?

Also, it would be nice to add a dependency: `useSelect( ..., [ window._currentSiteId ] )`

Reply:
> Could the store be referenced as import { Site } from '@automattic/data-stores' and Site.STORE_KEY?

Great idea.

> Also, it would be nice to add a dependency: useSelect( ..., [ window._currentSiteId ] )

But this is a global that will never change, and even if it didn't React won't know to rerender, is there any merit to passing it?